app/models/role.rb

```diff
@@ -78,6 +78,7 @@ class Role < ApplicationRecord
   scoped_search :on => :name, :complete_value => true
   scoped_search :on => :builtin, :complete_value => { :true => true, :false => false }
   scoped_search :on => :description, :complete_value => false
+  scoped_search :on => :id, :ext_method => :search_by_locked, :rename => :locked, :complete_value => { :true => true, :false => false }, :operators => ['= '], :only_explicit => true
```

Comment:
You should be able to do `:on => :locked` and drop the `:rename => :locked`

Reply:
works, thanks